content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: Todo lo que necesitas saber sobre fusionar datos en R
+author: Thomas Neitmann
+date: '2021-02-01'
+slug: r-merging-datasets
```

Comment:
Could you translate the `slug` as well? This will become the URL of the post.

Reply:
Done.